api/serializers/teacher_under_18.py

```diff
@@ -6,7 +6,7 @@
 class TeacherUnder18WriteSerializer(serializers.ModelSerializer[TeacherUnder18]):
     class Meta:
         model = TeacherUnder18
-        exclude = ("status",)
+        exclude = ()
```

Comment:
`Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disabled`
:)
That's why I left empty `exclude` in `Teacher` as well. 
I guess it's the equivalent of `fields = "__all__"`

Reply:
Yeah yeah, my bad. I think it's better to write `fields = "__all__"` since it's more common

Reply:
Done